config-specs/paper/server-properties.yml

```
@@ -167,6 +170,11 @@ rcon.password:
 rcon.port:
   default: "25575"
   description: "The port to start the rcon server on."
+region-file-compression:
+  default: "deflate"
+  description: >
+    Specifies the compression type used to compress region files. Possible values are: "deflate", "lz4" and "none".
+    If set to "none" region files will take up significantly more disk space but might make sense together with filesystem level compression.
```

```suggestion
    If set to "none", region files will take up significantly more disk space, but might make sense together with filesystem-level compression.
```

---

would this make more sense?
```suggestion
    If set to "none", region files will take up significantly more disk space, but it might make sense together with filesystem-level compression.
```

---

👍